src/SearchResults.js

```diff
@@ -0,0 +1,61 @@
+import React, { useState } from "react";
+import moment from "moment";
+const SearchResults = props => {
+  const [isSelected, setIsSelected] = useState([]);
```

Comment:
Clever approach and totally works. If you want to simplify it, try the steps I sent you on slack.